Adding a short description to contribute.md (#13730)
TheOdinProject/curriculum: javascript/organizing-js/objects-constructors.md

~~~javascript
const variable = 'property'

myObject.variable // this gives us 'undefined' because it's looking for a property named 'variable' in our object

> myObject[variable] // same as myObject['property'] and gives us 'Value!'

Reviewer: ```suggestion
myObject[variable] // this is equivalent to myObject['property'] and returns 'Value!'
```
